tests/trio200.py

```diff
@@ -0,0 +1,22 @@
+# ARGS --trio200-blocking-calls=bar->BAR,bee->NOERROR,bonnet->NOERROR,bee.bonnet->BEEBONNET
```

Comment:
is NOERROR special?

Reply:
or actually no that could in theory cause some other confusion/testing error since BEE and BONNET are substrings of BEEBONNET.
Maybe all magic words should be surrounded by underscores or something, not sure.